### PR TITLE
 Fix "This version" URL pointing to former source-map org

### PR DESCRIPTION
**Pull request by Krinkle**
The old URL serves 404 Error now.

### CODE REVIEW

File: source-map.bs [hunk 1]
```diff
@@ -4,7 +4,7 @@ H1: Source Map
 Shortname: source-map
 Level: 1
 Status: STAGE0
-URL: https://source-map.github.io/source-map-spec/
+URL: https://tc39.github.io/source-map-spec/
```
Comment (jkrems) [inline source-map.bs:7]: I think I would go all the way and resolve the redirects to https://tc39.es/source-map/.

**Review by jkrems — CHANGES_REQUESTED** [review]
Thanks for finding this!

File: source-map.bs (hunk 1 above)
Comment (Krinkle) [reply source-map.bs]: Okay.

**Review by jkrems — APPROVED** [review]

**Review by jkup — APPROVED** [review]

**Comment by jkup** [comment]
Thank you! Sorry this sat for so long, I was struggling to understand the IPR form check. Appreciate the PR!

**Pull request merged**